vpr/src/route/rr_graph.cpp

```diff
@@ -994,6 +997,7 @@ static void build_rr_graph(const t_graph_type graph_type,
                            const std::vector<t_physical_tile_type>& types,
                            const DeviceGrid& grid,
                            t_chan_width nodes_per_chan,
+                           t_router_opts router_opts,
```

Comment:
See my other comment on not passing in `router_opts`, but I thought I would mention this large oversite for you to look out for. `t_router_opts` is a massive structure. It should not be passed by value here and instead be passed by reference. Small mistakes like this can cause large degredations in performance. Always good to keep an eye out for, even though you should remove this for just the `route_verbosity` variable.